frontend/packages/pipelines-plugin/src/components/pipelineruns/list-page/PipelineRunRow.tsx

```diff
@@ -54,7 +56,7 @@ const PipelineRunRow: React.FC<RowFunctionArgs<PipelineRunKind>> = ({ obj, custo
       <TableData className={tableColumnClasses.name}>
         <ResourceLink
           kind={pipelinerunReference}
-          name={obj.metadata.name}
+          name={truncateMiddle(obj.metadata.name, { length: 12 })}
```

Comment:
The resource link `name` is also used as link! So the link tries to open:

`http://localhost:13469/k8s/ns/christoph/tekton.dev~v1~PipelineRun/nodein…uie93`

Which results for sure in a 404 Not found error when the user clicks on this link. :fearful: